pythonitalia/pycon
# Implement new design for the profile page

New profile page according to the design

The avatar and "My tickets" section is missing and we can implement it in another PR

# How to test

Open Vercel preview, login with your production account and see the new profile. Try editing and all

<img width="825" alt="image" src="https://user-images.githubusercontent.com/3382153/103182494-92852300-48a3-11eb-8279-84a58936a37b.png">

this PR already has the correct structure to implement the avatar section later

orders table:

(desktop)

<img width="1304" alt="image" src="https://user-images.githubusercontent.com/3382153/103213561-f6d8ce80-4905-11eb-8539-bcf6ce73786c.png">


(mobile)

<img width="515" alt="image" src="https://user-images.githubusercontent.com/3382153/103216635-6fdc2400-490e-11eb-9bc3-3fe173937617.png">



I also changed how My submission looks


This is in the profile page:

<img width="1234" alt="image" src="https://user-images.githubusercontent.com/3382153/103216592-563adc80-490e-11eb-9d30-b50cf73d8dde.png">


This is in the CFP page:

<img width="1342" alt="image" src="https://user-images.githubusercontent.com/3382153/103216611-605cdb00-490e-11eb-964f-ddb8882010f4.png">
